server/src/player-assignment/utils/getRunRandomAndPadgInput.ts

```diff
@@ -0,0 +1,71 @@
+import { getStartingGames } from 'server/player-assignment/utils/getStartingGames';
+import { getSignupWishes } from 'server/player-assignment/utils/getSignupWishes';
+import { getSignedGames } from 'server/player-assignment/utils/getSignedGames';
+import { getSelectedPlayers } from 'server/player-assignment/utils/getSelectedPlayers';
+import { getPlayerGroups } from 'server/player-assignment/utils/getPlayerGroups';
+import { getGroupMembers } from 'server/player-assignment/utils/getGroupMembers';
+import { User } from 'server/typings/user.typings';
+import { Game } from 'shared/typings/models/game';
+import { runRandomAndPadgInput } from 'server/typings/result.typings';
+
+export const getRunRandomAndPadgInput = (
```

Comment:
Tää on kyllä kiva, että näitä saa palasteltua vähän enemmän utilility-tiedostoihin.